recaf-ui/src/test/java/dev/xdark/recaf/TestUtils.java

```diff
@@ -0,0 +1,22 @@
+package dev.xdark.recaf;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Common test utilities.
+ */
+public class TestUtils {
```

Comment:
Same as other `TestUtils` comment. They can be shared from `core` version of `TestUtils`.